### ISSUE TITLE
Skrive infotekst om at kun de 100 mest aktive får utdelt lodd

### ISSUE THREAD

**Issue by xivind** [hdir/strava-club#49]
Siden dette er ganske permanent info kan det kanskje være nyttig å skrive det rett i markdown filene og ikke i variabelen for brukerdefinert tekst @thomiz ? Kan kanskje være nyttig å ha denne setningen i begge markdown filer, og ikke bare infofilen?

**Comment by thomiz** [comment]
Enig, dette er lagt til. Please review @xivind . Infoen er bare lagt til i docs/info.md

**Comment by xivind** [comment]
Teksten i docs/info.md ser bra ut @thomiz Men det kan hende at ikke alle får med seg den siden den ligger som et kulepunkt sammen øvrig info. Det er jo ganske viktig info, så jeg foreslår at vi i tillegg legger den på resultatsiden, se forslag til plassering i bildet under. Teksten kunne vært slik: 'Denne våren prøver vi ut om vi skal ha vår egen aktivitetskampanje i Helsedirektoratet. Det er foreløpig noen begrensninger i løsningen som gjør at dersom mer enn 100 personer oppnår 150 minutter aktivitet en uke, er det bare de 100 mest aktive som får tildelt lodd for den uken.' Dersom du er enig, gi beskjed, så kan jeg fikse det, og dersom du tenker det er godt nok som det er, bare lukk saken.



![Image](https://github.com/hdir/strava-club/assets/8987107/24d94526-2516-4512-91bf-e40fecafd8ec)

**Comment by thomiz** [comment]
Jeg er usikker, tror det avhenger veldig av hvor mange som faktisk blir aktive deltakere i aksjonen. Hvis det blir mange deltakere bør vi legge denne informasjonen inn på forsiden.

**Comment by xivind** [comment]
Det er en god vurdering @thomiz . Jeg er enig. Da stenger jeg denne nå, så løfter vi det opp på fremsiden senere, om vi ser at vi blir over 100 aktive.

**Comment by thomiz** [comment]
Jeg tror vi kan løse dette med å splitte opp i to klubber og skrape begge, vi har allerede testet denne funksjonen @xivind 
[TestBuild](https://hdir.github.io/strava-club/TestBuild/results.html). Det betyr at teksten kanskje skal endres til at medlemsmassen må fordele seg på to klubber for å sikre at alle resultater kommer med?

**Comment by xivind** [comment]
Ja, det stemmer @thomiz Jeg husker det nå! Når du testet med TCK. I så fall er vel kun nødvendig med endringer i scraperen? Teksten må helt klart endres for å få dette frem. Kan vi ikke diskutere dette kort etterpå i dag, hvordan vi kommuniserer dette? Vi må være klare, men vi trenger vel kanskje først å kommunisere denne muligheten når vi f. eks passerer 80?
